sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql

```diff
@@ -0,0 +1,23 @@
+-- Add column note_id to sims.disbursement_overawards.
+ALTER TABLE
+    sims.disbursement_overawards
+ADD
+    COLUMN IF NOT EXISTS note_id INTEGER REFERENCES sims.notes(id);
```

Comment:
I missed the same on the previous PR, but most of the time I believe that we use the types like `INT`.